WebRoot/js/zimbraMail/calendar/view/ZmApptEditView.js

```diff
@@ -2216,13 +2224,15 @@ function() {
 
 	this._allDayCheckbox._editViewId = this._repeatDescField._editViewId = edvId;
 	this._startDateField._editViewId = this._endDateField._editViewId = edvId;
-    if(this._showOptional) this._showOptional._editViewId = edvId;
-    if(this._showResources) this._showResources._editViewId = edvId;
-    if (this.GROUP_CALENDAR_ENABLED) {
-        this._suggestTime._editViewId = edvId;
-    }
-    this._suggestLocation._editViewId = edvId;
-    this._locationStatusAction._editViewId = edvId;
+	if(this._showOptional) this._showOptional._editViewId = edvId;
```

Comment:
small, better to have if  blocks consistent , either use curly braces or not (for one liner if blocks).

Reply:
Fixed: https://github.com/Zimbra/zm-web-client/pull/143/commits/b15e5e1cd57ec3595682a88031056b2ef693148d#diff-4ecbe8e20f6ce7247203cc12d861f949R2227

Thanks